src/operator/account/AccountGrid.tsx

```diff
@@ -47,6 +51,15 @@ const AccountGrid: FC = () => {
           body={organizations?.[0]?.provider ?? 'N/A'}
           testID="cloud-provider"
         />
+        <AccountField
+          header="Cancelled At"
+          body={
+            account.type === 'cancelled' && cancelledAt
+              ? `${cancelledAt.toLocaleTimeString()} ${cancelledAt.toDateString()}`
+              : 'N/A'
+          }
+          testID="cancelled-at"
+        />
       </FlexBox>
```

Comment:
Looks good; but if you change anything based on the above comment, this component should have the same logic. 👍